src/OrderFormAcceptanceTests.Steps/Steps/CompleteOrder.cs

```diff
@@ -16,12 +16,19 @@ public CompleteOrder(UITest test, ScenarioContext context) : base(test, context)
 
         }
 
+        [Given(@"the User chooses to complete the Order")]
         [When(@"the User chooses to complete the Order")]
```

Comment:
Since this Given and When are the same, change the keyword to StepDefinition which is generic and will map to all step types, then remove the duplicate.

Reply:
missed this one?